src/Chapter3.hs

```diff
@@ -907,9 +1108,27 @@ Implement instances of "Append" for the following types:
   ✧ *(Challenge): "Maybe" where append is appending of values inside "Just" constructors
 
 -}
+data Gold = Gold
+  { gold  :: Int
+  } deriving (Show)
+
 class Append a where
     append :: a -> a -> a
 
+instance Append Gold where
+    append :: Gold -> Gold -> Gold
+    append x y = Gold((gold x) + (gold y))
```

Comment:
parenthesis are redundant for the function applications 🙂 
Also, in Haskell, there are spaces between functions and arguments, even if there are parenthesis there 👾 
```suggestion
    append x y = Gold (gold x + gold y)
```